feature/SALOONY-010-reset-password

hazzem-badran opened the pull request.

MohamedEllouze commented on components/organisms/reset-form/style.ts:
```diff
@@ -0,0 +1,27 @@
+    marginTop: 5,
+  },
+  label: {
+    fontSize: 18,
```
for the font size fix the main font size with 16px then use rem and em in the rest of app

hazzem-badran replied:
عملت التعديلات اللي كانت موجودة بالبرانشات التانية بس هاي حبب البرانشات اتمر عشان اعمل برانش وبمكان واحد ويصر على الكل بالكامل
+ اني بدي اعمل تحسيات على انبت والستايل تاعو ييعني اخليه متابق لفقما وتحديث مكتبات المشروع

MohamedEllouze commented:
Resolve conflict

hazzem-badran committed:
feature/SALOONY-010-reset-password

# Conflicts:
#	components/organisms/login-form/index.tsx
#	components/organisms/reset-form/style.ts
#	constants/formFields.constants.ts
#	types/forms.ts
